documentation/milestones/milestone_template.md

```diff
@@ -45,50 +88,37 @@ The following work will *not* be completed as part of this milestone:
 ## Integrations
 
 ### Translations
+<!-- Optional -->
```

Comment:
Hmm, I'm chewing on whether these questions - Translations, services to PROD, etc - should be optional or not. They could be required but then have easy answers like "no".

Reply:
Likewise, updated all of the items in this section to required.